src/vttLib/transfer.py

```diff
@@ -10,29 +10,43 @@
 logger = logging.Logger(__name__)
 
 
-def dump_to_file(font: fontTools.ttLib.TTFont, path: os.PathLike):
+def dump_to_file(font: fontTools.ttLib.TTFont, path: os.PathLike) -> None:
     """Dump relevant VTT data to a file.
 
     Relevant are:
     - TSI1 holds the assembly code
     - TSI3 holds the VTT code
     - TSI5 holds glyph group information
+    - TSIC holds data for the cvar table BUT IS NOT A STRICT SUPERSET
```

Comment:
Do you have an example of what is in cvar but not TSIC, for reference in the future?

Reply:
What I found with Stroudley UI is that there was data in cvar that was not in TSIC (deleting cvar and recompiling TSIC in VTT compiled a smaller cvar). So theoretically both can contain the same data, but as of VTT 6.33 Beta they probably don't?